abstract/test.js

```diff
@@ -0,0 +1,20 @@
+'use strict'
+
+module.exports = function (test, level, options) {
```

Comment:
In a new major this could be renamed to e.g. `all.js` and options can be removed.